app/models/planning_application.rb

```diff
@@ -40,6 +41,28 @@ def reference
     @_reference ||= id.to_s.rjust(8, "0")
   end
 
+  def correction_provided?
+    awaiting_determination? && reviewer_decision&.private_comment.present?
+  end
+
+  def reviewer_disagrees_with_assessor?
+    return false unless reviewer_decision && assessor_decision
+
+    reviewer_decision.status != assessor_decision.status
+  end
+
+  def assessor_decision_updated?
+    return false unless assessor_decision && reviewer_decision
+
+    assessor_decision.updated_at >= reviewer_decision.updated_at
```

Comment:
Why have you used >=  while the reviewer_decision_updated is simply > ?

Reply:
Good spot, I might have forgot it